src/Components/CreateImageWizard/CreateImageWizard.js

```diff
@@ -516,6 +518,8 @@ const requestToState = (composeRequest, distroInfo, isProd, enableOscap) => {
     if (enableOscap) {
       formState['oscap-profile'] =
         composeRequest?.customizations?.openscap?.profile_id;
+      composeRequest?.customizations?.openscap?.profile_name;
```

Comment:
Correct me if I'm wrong, but this is not really forming a state, right?

Shouldn't there be an assignment to the state?
```suggestion
      formState['oscap-profile-name'] = composeRequest?.customizations?.openscap?.profile_name;
```